### PR TITLE
Mapping Search Placeholder changed

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**
Changed the placeholder from Search to Search For More

### CODE REVIEW

File: src/app/shared/components/mapping/mapping-table/mapping-table.component.html [hunk 1]
```diff
@@ -16,8 +16,9 @@ <h4>{{ this.destinationType | titlecase | snakeCaseToSpaceCase }} in QuickBooks
     <td mat-cell *matCellDef="let element">
       <mat-form-field floatLabel="always" appearance="outline" class="mapping-table--form-field">
         <mat-select placeholder="Select {{ this.destinationType | titlecase | snakeCaseToSpaceCase }}" [value]="mappingForm[element.index]?.value?.destination" (closed)="helperService.clearSearchText(mappingForm[element.index])">
-          <app-simple-text-search [form]="mappingForm[element.index]"  [isSearchInProgress]="isSearchInProgress" [page]="SimpleSearchPage.MAPPING" [advancedSearchType]="SearchType.SELECT_FIELD" (searchResult)="advancedSearchHandler($event)"></app-simple-text-search>
-
+          <div fxLayout="row">
```
Comment (ashwin1111) [inline src/app/shared/components/mapping/mapping-table/mapping-table.component.html:19]: remove line 19 and 21

File: src/app/shared/components/mapping/mapping-table/mapping-table.component.ts [hunk 2]
```diff
@@ -106,8 +108,15 @@ export class MappingTableComponent implements OnInit {
...
 
+  titleCase(str: string): string {
+    return str.toLowerCase().replace(/(?:^|\s|-)\w/g, (match) => {
+      return match.toUpperCase();
+    });
+  }
+
   ngOnInit(): void {
     this.existingQboOptions = this.qboData.concat();
+    this.toolTipContent = this.sourceType === FyleField.CATEGORY ? this.titleCase('If you are unable to find an account in the dropdown, please use the search bar to find from your Chart of Accounts') : this.titleCase(`If you are enable to find a ${this.destinationType} in the dropdown please use the search bar to find from your ${this.destinationType}s`)
```
Comment (ashwin1111) [inline src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: fix linting error

Comment (Ashutosh619-sudo) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: is there a library should i be using to always lint

Comment (Ashutosh619-sudo) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: is there a library should i be using to always lint.

Comment (Ashutosh619-sudo) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: is there a library should i be using to always lint.

Comment (Ashutosh619-sudo) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: is there a library should i be using to always lint.

Comment (Ashutosh619-sudo) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: is there a library should i be using to always lint.

Comment (ashwin1111) [reply src/app/shared/components/mapping/mapping-table/mapping-table.component.ts:119]: you can do `ng lint --fix`, this will auto fix most of the things

**Pull request merged**